app/instrumentation-tests/src/org/commcare/androidTests/FixturesTest.kt

```diff
@@ -0,0 +1,166 @@
+package org.commcare.androidTests
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.doesNotExist
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import org.commcare.dalvik.R
+import org.commcare.utils.CustomMatchers
+import org.commcare.utils.InstrumentationUtility
+import org.hamcrest.Matchers.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class FixturesTest: BaseTest() {
+
+    companion object {
+        const val CCZ_NAME = "fixtures.ccz"
+        const val APP_NAME = "Fixtures"
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+    }
+
+    @Test
+    fun testMissingFixture_throwsError() {
+        InstrumentationUtility.login("fixtures_fails", "123")
+        openFixtureForm()
+        onView(withId(R.id.nav_btn_next))
+                .perform(click())
+        onView(withText("Error Occurred"))
+                .check(matches(isDisplayed()))
+        onView(withText(startsWith("Make sure the 'Test' lookup table is available")))
+                .check(matches(isDisplayed()))
+    }
+
+    @Test
+    fun testFixturesWork() {
+        InstrumentationUtility.login("fixtures_works", "123")
+        openFixtureForm_andSelectFirstTwoCheckboxes()
+        onView(withText("Essex"))
+                .perform(click())
+        onView(withId(R.id.nav_btn_next))
```

Comment:
both next and prev navigations can be encapsulated into functions so that if we ever change the view id of buttons, we only need to change the ui tests in one place.

Reply:
I think it goes with all the strings/ids that we're using in androidTests and it's always hard to have them at one place. So I feel like not doing it.